css/properties/writing-mode.json

```
@@ -171,6 +171,54 @@
               "deprecated": true
             }
           }
+        },
+        "vertical_oriented_form_controls": {
+          "__compat": {
+            "description": "Vertically-oriented form control support",
```

Sorry I mean to remove "support" here, too. BCD is always about support.

```suggestion
            "description": "Vertically-oriented form controls",
```